pkg/skaffold/kubernetes/log.go

```diff
@@ -240,6 +240,12 @@ func (l *ImageList) AddImage(image string) {
 	l.Unlock()
 }
 
+func (l *ImageList) RemoveImage(image string) {
+	l.Lock()
+	l.names[image] = false
```

Comment:
should we remove it from the list instead of setting to false? It looks like thats how the trckedContainers type handles that above.

Reply:
done